.github/workflows/test_dbt_models.yaml

```diff
@@ -3,7 +3,19 @@ name: test-dbt-models
 # This workflow is not scheduled or run on PRs because it is manually triggered
 # by the Data Department's sqoop data extraction process upon completion. See
 # https://github.com/ccao-data/service-sqoop-iasworld
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      select:
+        description: dbt select statement for tests to run
+        required: false
+        default: --selector select_data_test_iasworld
```

Comment:
It's annoying that this design requires us to include the option name, but I think it's simpler to merge the `--select` and `--selector` at this level of abstraction rather than requiring the workflow caller to handle two separate variables. I'm open to other design suggestions if you have them, though!

Reply:
Good point! I realized that the `--selector` argument is unnecessary anyway, since we only have one selector (`select_data_test_iasworld`) and we can just default to it if the `select` argument is empty. If we ever add more selectors it should be easy to add support for a `--selector` argument here, but in the meantime, one input variable that overrides the default selector with a custom `--select` statement feels a lot clearer to me.